CMakeLists.txt

```diff
@@ -129,10 +129,12 @@ if(BUILD_BENCHMARK)
   add_subdirectory(benchmark)
 endif()
 
-if(BUILD_PYTHON)
```

Comment:
It looks like the wrappers should be up to date. I might have forgot to push this change